hooks/post_gen_project.sh

```diff
@@ -3,8 +3,7 @@
 set -e
 
 {% if cookiecutter.fabric == "Yes" %}
-carthage update --platform iOS
-echo "✅ Fabric frameworks were added successfully. Don't forget to add API Key to Info.plist and API Key and Build Secret to Fabric Run Script Build Phase. The credentials are available here: https://www.fabric.io/settings/organizations"
+echo "⚠️ Don't forget to add API Key to Info.plist and API Key and Build Secret to Fabric Run Script Build Phase. The credentials are available here: https://www.fabric.io/settings/organizations"
```

Comment:
New projects will not presented in Fabric Beta

Reply:
Thanks for the comment! I've added a task to command backlog.